scripts/managed-services/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function main() {
+  local kafkaId=$1
+  local managedKafkaCR=$(oc -n kafka-$kafkaId get managedkafka -o name)
+  local kafkaCR=$(oc -n kafka-$kafkaId get kafka -o name)
+  local kafkaStatefulSet=$(oc -n kafka-$kafkaId get statefulset -l app.kubernetes.io/name=kafka -o name)
+  local timestamp=$(date +%s)
+  local inspectDir="inspect.kafka-$kafkaId.$timestamp"
+  local namespaceCheck=$(oc get namespace kafka-$kafkaId)
+
+  oc -n kafka-$kafkaId adm inspect ns/kafka-$kafkaId $managedKafkaCR $kafkaCR --dest-dir $inspectDir > /dev/null 2>&1
```

Comment:
Don't assume that kafka namespace will always be prefixed by 'kafka-'.  Just accept the fully namespace as an argument to the script.

Reply:
We don't what the output to include the kubermetes secrets.  AFAIK you can't tell `oc adm` not to get secrets, so just prune the secrets directory afterwards:
```
find ${inspectDir} -type d -name secrets -delete
```